examples/boids/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "wgpu-boids-example"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+description = "wgpu boids example"
+homepage.workspace = true
+repository.workspace = true
+keywords.workspace = true
+license.workspace = true
+publish = false
```

Comment:
Do we need all these fields? since we won't publish those

Reply:
Good shout, will trim them down

Reply:
Had to keep license, version, and edition, the rest (of the .workspace ones) I could rid.